### DISCUSSION TITLE
Cursor speed option (Android)

### DISCUSSION THREAD

**Discussion by gudetamagang** [ppy/osu#16538] [Ideas] [answered]
I think having a cursor speed option on the Android version would be nice for using a mouse or tablet. I'm not sure how difficult this would be to implement, seeing as most tablets on android map the entire area to the screen, but because many tablets on android don't have a mapping tool (I think only xp-pen has one), being able to change the cursor speed would be great since playing with such a low speed makes it impossible to play with a tablet or even a mouse, at least in terms of how you would play on PC.

**Comment by peppy** [comment, marked as answer]
This will be supported in the near future (see https://github.com/ppy/osu-framework/pull/4971)
